src/scss/language-processing.scss

```diff
@@ -138,6 +138,82 @@
     }
 }
 
+.media-frame-content {
+	.generated-images,
+	.prompt-view {
+		padding: 0 16px
+	}
+
+	.generated-images {
+		ul {
+			display: flex;
+			flex-wrap: wrap;
+			list-style-type: none;
+			gap: 18px;
+			margin: 0;
+			padding: 0;
+		}
+
+		li {
+			margin-bottom: 20px;
+		}
+
+		.generated-image {
+			img {
+				box-shadow: inset 0 0 15px rgb(0 0 0 / 10%), inset 0 0 0 1px rgb(0 0 0 / 5%);
+				border: 1px solid rgb(0 0 0 / 10%);
+				background: #f0f0f1;
+				display: block;
+				margin-bottom: 10px;
+				width: 460px;
+				height: 460px;
+			}
+
+			button {
+				display: block;
+				float: left;
+				margin-right: 10px;
+			}
+
+			.spinner {
+				float: left;
+				margin: 6px;
+			}
+		}
+
+		.spinner {
+			float: none;
+			visibility: hidden;
+
+			&.active {
+				visibility: visible;
+			}
+		}
+
+		.error {
+			color: #cf2e2e;
+			font-size: 15px;
+		}
+	}
+
+	.prompt-view {
+		margin-top: 20px;
+
+		.prompt {
+			margin-right: 10px;
+			padding: 1px 8px;
+			width: 25%;
```

Comment:
Here i can see width is 25% but not seeing anything related to media queries, Are we not considering mobile versions?